test/bvt/sqlsrv/msdn_sqlsrv_cancel.phpt

```diff
@@ -46,4 +46,4 @@ echo "$count sales accounted for the first $$salesTotal in revenue.\n";
 sqlsrv_cancel( $stmt);
 ?>
 --EXPECT--
-58 sales accounted for the first $104205.9607 in revenue.
\ No newline at end of file
+57 sales accounted for the first $104171.7607 in revenue.
```

Comment:
It's the problem with the sql query itself. See #436 for details. Back then I put the incorrect expected output.